generators/client/templates/angular/src/main/webapp/app/shared/tracker/_tracker.service.ts

```diff
@@ -11,6 +10,9 @@ import { CSRFService } from '../auth/csrf.service';
 import { AuthServerProvider } from '../auth/auth-jwt.service';
 <%_ } _%>
 
+import SockJS = require('sockjs-client');
```

Comment:
In theory, the theory and the practice is the same, but in practice ... :wink: 
If you write:

```
import SockJS from 'sockjs-client';
const socket = new SockJS(url);
```
It will translated to
```
var sockjs_client_1 = __webpack_require__(1056);
var socket = new sockjs_client_1.default(url);
```
If you write:
```
import { SockJS } from 'sockjs-client';
const socket = new SockJS(url);
```
it is translated to :
```
var sockjs_client_1 = __webpack_require__(1056);
var socket = new sockjs_client_1.SockJS(url);
```

and of course, neither is ok. So if you know a better option, I'm open to it, but unfortunately I haven't found one.